src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -152,16 +161,21 @@ def _expiry(self, expiry):
 class CookieInformation(object):
 
     def __init__(self, name, value, path=None, domain=None, secure=False,
-                 httpOnly=False, expiry=None):
+                 httpOnly=False, expiry=None, **extra):
         self.name = name
         self.value = value
         self.path = path
         self.domain = domain
         self.secure = secure
         self.httpOnly = httpOnly
         self.expiry = datetime.fromtimestamp(expiry) if expiry else None
+        if extra:
+            self.extra = extra
```

Comment:
Conditionally setting or not setting attribute is a bit of an anti-pattern. Is there some benefit here?

Reply:
That is true. Although I did explain it in the docs. Will change.